### PR TITLE
CORS 'Origin' header guidance

### LINKED ISSUE #14276: Does NOT work behind load balancers
I have ASP.Net Core 2.2 deployed on Kubernetes cluster behind load balancer. CORS does NOT work behind load balancer. Client application does not receive the CORS response headers from the server application. Any advice and insight is appreciated.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 59bca5db-1196-2a8b-a0d8-4e89b2382288
* Version Independent ID: d3f332c4-4c60-039e-0e1e-bc6b11831f87
* Content: [Enable Cross-Origin Requests (CORS) in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/cors?view=aspnetcore-2.2)
* Content Source: [aspnetcore/security/cors.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/security/cors.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by guardrex**
Fixes #14276

Thanks @khteh! :rocket:

### CODE REVIEW

File: aspnetcore/security/cors.md [hunk 1]
```diff
@@ -487,7 +487,7 @@ To test CORS:
 
      **Access to XMLHttpRequest at `https://webapi.azurewebsites.net/api/values/1` from origin `https://localhost:44375` has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.**
      
-CORS-enabled endpoints can be tested with a tool, such as [Fiddler](https://www.telerik.com/fiddler) or [Postman](https://www.getpostman.com/). When using a tool, the origin of the request specififed by the `Origin` header must differ from the host receiving the request. When the `Origin` header isn't present or indicates the same host, the CORS Middleware assumes that the request is from the same origin. Since the request isn't *cross-origin*:
+CORS-enabled endpoints can be tested with a tool, such as [Fiddler](https://www.telerik.com/fiddler) or [Postman](https://www.getpostman.com/). When using a tool, the origin of the request specififed by the `Origin` header must differ from the host receiving the request. If the request isn't *cross-origin* based on the value of the the `Origin` header:
```
Comment (khteh) [inline aspnetcore/security/cors.md]: Duplicate "the the"

Comment (guardrex) [reply aspnetcore/security/cors.md]: ooooops ..... this happens more than you'd think. When looking at these in an editor (most of us use VS Code) and the line breaks right in the middle of the double-word, it's *hard to spot*. Thanks for catching :eyes: that!

Comment (guardrex) [reply aspnetcore/security/cors.md]:
```suggestion
CORS-enabled endpoints can be tested with a tool, such as [Fiddler](https://www.telerik.com/fiddler) or [Postman](https://www.getpostman.com/). When using a tool, the origin of the request specififed by the `Origin` header must differ from the host receiving the request. If the request isn't *cross-origin* based on the value of the `Origin` header:
```

Comment (guardrex) [reply aspnetcore/security/cors.md]: Jerks!!! lol ..... they won't let me commit my own suggestion. 😠

**Pull request merged**